src/videoCache.js

```diff
@@ -124,3 +124,7 @@ export function store(bids, done) {
     withCredentials: true
   });
 }
+
+export function getCacheUrl(id) {
+  return `${PUT_URL}?uuid=${id}`;
```

Comment:
This should be renamed to something like `base_url` or similar because `PUT` is confusing in this context (get op).